### PR TITLE
JIRA-25 Enable https on client side

### PR DESCRIPTION
**Pull request by peterholman**
Adds support for .env files and enables https by default. 

Would be worth pulling the branch and checking the new run scripts in `package.json` work for you. You'll need to follow the new steps in the `README.md`.

If you want to setup the certs stuff locally you can, alternatively you can just ignore the security warning when you navigate to the client in the browser.

### CODE REVIEW

File: client/.env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# default environment variables, always imported
+PORT=3030
+HTTPS=true
```
Comment (peterholman) [inline client/.env:3]: this enables https by default when you spin up the client

File: client/environment/template.env.local
Comment (peterholman) [inline client/environment/template.env.local:1]: you don't want to push the .env.local file to the remote as it is used for local configuration. I've created a template version of the file which can be used to get things running locally. I've updated the `README.md` with instructions.

File: client/environment/.env.dev
Comment (peterholman) [inline client/environment/.env.dev:1]: This is a placeholder file which will be used for providing configuration for the dev environment which will be running on the Oracle VM.

File: client/package.json [hunk 2]
```diff
@@ -30,7 +31,8 @@
     "typescript-eslint": "^7.10.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start:local": "env-cmd -f environment/.env.local react-scripts start",
```
Comment (peterholman) [inline client/package.json:34]: These scripts are used to pull in the environment variables from the `*.env` files I've added.

**Review by MikesCS50 — APPROVED** [review]

**Pull request merged**